GitCommands/Logging/CommandLogger.cs

```diff
@@ -6,11 +6,10 @@ namespace GitCommands.Logging
 {
     public sealed class CommandLogger
     {
-        public delegate void CommandsChangedHandler();
         private const int LogLimit = 500;
         private readonly Queue<CommandLogEntry> _logQueue = new Queue<CommandLogEntry>(LogLimit);
 
-        public event CommandsChangedHandler CommandsChanged = delegate { };
+        public event EventHandler CommandsChanged = delegate { };
```

Comment:
:bulb: I understand why it's assigning a default value of `delegate { }` (so you never need to check for null at the invocation site), but I don't like it because it introduces performance overhead for little (if any) gain in clarity.